### ISSUE TITLE
Babel-loader pls move dependency do devdependency

### ISSUE THREAD

**Issue by dpgmiku** [kybarg/react-barcode-reader#19]
Hi,

**tl;dr** can you pls move babel-loader from dependency to devDependency? Thank you!

When Im trying to run my project with you library and react-scripts 3.4.0 (we dont use 3.4.1 because there is a bug not fixed for test coverage), we are getting this error:

`To fix the dependency tree, try following the steps below in the exact order:

  1. Delete package-lock.json (not package.json!) and/or yarn.lock in your project folder.
  2. Delete node_modules in your project folder.
  3. Remove "babel-loader" from dependencies and/or devDependencies in the package.json file in your project folder.
  4. Run npm install or yarn, depending on the package manager you use.

In most cases, this should be enough to fix the problem.
If this has not helped, there are a few other things you can try:

  5. If you used npm, install yarn (http://yarnpkg.com/) and repeat the above steps with it instead.
     This may help because npm has known issues with package hoisting which may get resolved in future versions.

  6. Check if /home/mkubacki/workspace/team-cat/backstage/packages/app/node_modules/babel-loader is outside your project directory.
     For example, you might have accidentally installed something in your home folder.

  7. Try running npm ls babel-loader in your project folder.
     This will tell you which other package (apart from the expected react-scripts) installed babel-loader.

If nothing else helps, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That would permanently disable this preflight check in case you want to proceed anyway.

P.S. We know this message is long but please read the steps above :-) We hope you find them helpful!
`

I tried this steps but didnt work. If we move babel to devdependency, it will work as far as I know. I would love to get your support.

**Comment by mjwvb** [comment]
I'm on react-script v4.0.3, and having this problem as well.
Fixed it for now by forcing babel-loader 8.1.0 in package.json:

"dependencies": {
  "babel-loader": "8.1.0"
}
